ansible/roles/dokuwiki/tasks/main.yml

```diff
@@ -2,10 +2,44 @@
 - include: tasks/docker.yml
 
 - name: Create supervisor config
-  template: src=supervisor/dokuwiki.conf
-            dest=/etc/supervisor/conf.d/dokuwiki.conf
+  copy:
+    src: supervisor/dokuwiki.conf
+    dest: /etc/supervisor/conf.d/dokuwiki.conf
 
 - name: Create nginx config
   template:
     src: nginx/dokuwiki.conf
     dest: /etc/nginx/sites-enabled/dokuwiki.conf
+
+- name: Create the initialization script
+  copy:
+    src: initialize.d/dokuwiki
+    dest: /opt/common/initialize.d/dokuwiki
+    mode: 0755
+
+- name: Subscribe to hooks
+  copy:
+    src: hooks/user
+    dest: /opt/common/hooks/{{ item }}.d/dokuwiki
+    mode: 0755
+  with_items:
+    - user-created
+    - user-passwd
+    - user-deleted
+
+- name: Create the libexec folder
+  file:
+    path: /opt/dokuwiki/libexec
+    state: directory
+    mode: 0755
+
+- name: Install update_user script
+  copy:
+    src: libexec/update_user
+    dest: /opt/dokuwiki/libexec/update_user
+    mode: 0755
+
+- name: Allow liquid user to call update_user with sudo
```

Comment:
See above, move the `sudoers.d` part in a `file` under `common` that adds the `invoke hook` script for user liquid.